examples/sort/parallel_sort.py

```diff
@@ -0,0 +1,77 @@
+import ray
+
+import numpy as np
+import time
+
+num_cores = 6
+num_partition_blocks = num_cores
+num_sorting_blocks = num_cores
+num_samples_for_pivots = num_partition_blocks * 25
+array_len = 100000000
```

Comment:
I'd suggest using notation like `10**8` so people don't have to count the zeros.